components/http-accept.js

```diff
@@ -4,13 +4,12 @@ var _ = require('underscore');
 var getRawBody = require('raw-body');
 var typer = require('media-typer');
 
-var promiseOutput = require('../src/promise-output');
-var componentFactory = require('../src/noflo-component-factory');
+var wrapper = require('../src/javascript-wrapper.js');
 
-exports.getComponent = componentFactory({
+module.exports = wrapper({
     description: "Extracts the request body as a String and produces a 202 Accepted response. Optionally, also filters on request content type.",
     icon: 'sign-in',
-    outPorts: _.defaults({
+    outPorts: {
         rejected: {
```

Comment:
No, this is not used to transmit state, but to provide the HTTP response back over HTTP.